enumfields/compat.py

```diff
@@ -65,3 +65,19 @@ def import_string(dotted_path):
         module_path, class_name = dotted_path.rsplit('.', 1)
         module = import_module(module_path)
         return getattr(module, class_name)
+
+
+def import_class(components):
```

Comment:
@akx Sorry for my late reply. This is created to support "Enums defined as nested classes now can be migrated". An example for this is

```python
# module: demo.hello.models

from django.db import models
from enumfields import IntEnum, EnumIntegerField

class Order(models.Model):
    class State(IntEnum):
        PENDING, ERROR, DONE = xrange(3)

    state = EnumIntegerField(State, default=State.PENDING)
```


As I found it's sometimes more elegant to define an enum as a nested class of a Django Model class (in that case, it's `Order.State`). So in this case, the `enum` parameter passed to `EnumField` is a `tuple` of 2 strings: `('demo.hello.models', 'Order.State')`, and it will also work with deeper-nested classes...

Reference: https://github.com/hzdg/django-enumfields/pull/68/files#diff-0d0002b8d572eae99755278bc4aeb328R110